CMakeLists.txt

```diff
@@ -209,15 +210,13 @@ if (WIN32)
     "/EHsc"
   )
   add_definitions(-DMS_WINDOWS)  # Globally define MS_WINDOWS
-  else()
+else()
   add_compile_options(
     "-Wno-c99-designator"         # Check if still necessary?
     "-Wno-return-type-c-linkage"  # Check if still necessary?
     "-Wno-macro-redefined"        # Check if still necessary?
     "-Wno-implicit-const-int-float-conversion" # Necessary for DataSketches
-    "-fwrapv"                     # Conda used to force-add. Remove in follow up PR
-    "-fstack-protector-strong"    # Conda used to force-add. Remove in follow up PR
-    "-D_FORTIFY_SOURCE=2"         # Conda used to force-add. Remove in follow up PR
+    "-fwrapv"                     # Conda used to force-add. Its an optimization for integer overflow
```

Comment:
The conda-forge team wants us to just use the flags that they set in their environment, instead of overriding it

Reply:
I think because conda-forge changes stuff on their end, so we can let them decide. Plus, they generally make Bodo slower, so we should avoid if possible